test/integration/framework/jsonify_benchmark.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	benchparse "golang.org/x/tools/benchmark/parse"
+	"k8s.io/kubernetes/test/e2e/perftype"
+)
+
+func main() {
+	err := run()
+	if err != nil {
+		panic(err)
+	}
+}
+
+func run() error {
+	if len(os.Args) < 2 {
+		return fmt.Errorf("output filename is a required argument")
+	}
+	benchmarkSet, err := benchparse.ParseSet(os.Stdin)
+	if err != nil {
+		return err
+	}
+	data := perftype.PerfData{Version: "v1"}
+	for _, benchMarks := range benchmarkSet {
+		for _, benchMark := range benchMarks {
```

Comment:
Also add the metric for no. of iterations? It can potentially help in some variance calculations.